packages/aws-cdk-lib/aws-cloudfront-origins/test/s3-bucket-origin.test.ts

```diff
@@ -466,6 +466,72 @@ describe('S3BucketOrigin', () => {
         });
       });
     });
+    describe('when specifying READ, WRITE, and DELETE origin access levels', () => {
+      it('should add the correct permissions to bucket policy', () => {
```

Comment:
I expected this case to have been covered by the existing unit test: https://github.com/samson-keung/aws-cdk/blob/4c9d7196370351c9f4f7a61b82e2981520b0f516/packages/aws-cdk-lib/aws-cloudfront-origins/test/s3-bucket-origin.test.ts#L20

Or do we need this one to focus on the security aspect?

Reply:
Yes I was thinking this is scoped to checking the permissions and covers `DELETE` permissions as well.

Reply:
I would suggest modifying the existing unit test to cover the DELETE case so we do not have overlapping tests. But with that said, I don't think it is a blocked to have overlapping tests either.